apps/frontend-manage/src/components/common/SupportModal.tsx

```diff
@@ -0,0 +1,166 @@
+            <p className="mb-4 leading-6 prose">
+              {t('manage.support.feedbackText')}
+            </p>
+            <div className="flex flex-col gap-2">
+              <SupportEntry
+                href="https://klicker-uzh.feedbear.com/boards/feature-requests"
+                title={t('manage.support.featureRequest')}
+                subtitle={t('manage.support.featureRequestDesc')}
+                icon={faLightbulb}
+                data={{ cy: 'support-feature-request' }}
+              />
+              <SupportEntry
+                href="https://klicker-uzh.feedbear.com/boards/bug-reports"
+                title={t('manage.support.bugReport')}
+                subtitle={t('manage.support.bugReportDesc')}
+                icon={faBug}
+                data={{ cy: 'support-bug-report' }}
+              />
+              <SupportEntry
+                href="https://klicker-uzh.feedbear.com/boards/self-hosting"
+                title={t('manage.support.selfHosting')}
+                subtitle={t('manage.support.selfHostingDesc')}
+                icon={faServer}
+                data={{ cy: 'support-self-hosting' }}
+              />
+            </div>
+          </div>
+
+          <div className="order-1 md:order-2 w-[200px] py-8 md:p-0 m-auto md:m-0">
+            <Image
+              src="/KlickerLogo.png"
+              width={300}
+              height={90}
+              alt="KlickerUZH Logo"
+              className="mx-auto"
+              data-cy="login-logo"
+            />
+          </div>
+        </div>
+
+        <div className="flex flex-col justify-between flex-1">
+          <div className="flex flex-col gap-2">
+            <H2>{t('manage.support.furtherResources')}</H2>
+            <div className="text-gray-600">
+              {t('shared.generic.documentation')}
+            </div>
+            <SupportEntry
+              href="https://www.klicker.uzh.ch/getting_started/welcome"
+              title={t('shared.generic.documentation')}
+              subtitle={t('manage.support.documentationDesc')}
+              icon={faBook}
+              data={{ cy: 'support-documentation' }}
+            />
+            <SupportEntry
+              href="https://www.klicker.uzh.ch/faq"
+              title={t('manage.support.faq')}
+              subtitle={t('manage.support.faqDesc')}
+              icon={faQuestion}
+              data={{ cy: 'support-faq' }}
+            />
+            <div className="mt-4 text-gray-600">
+              {t('manage.support.connect')}
+            </div>
+            <SupportEntry
+              href="https://www.klicker.uzh.ch/community"
+              title={t('manage.support.community')}
+              subtitle={t('manage.support.communityDesc')}
+              icon={faComments}
+              data={{ cy: 'support-community' }}
+            />
+            {user?.catalyst ? (
+              <SupportEntry
+                href="mailto:klicker.support@uzh.ch"
```

Comment:
```suggestion
                href="mailto:klicker@bf.uzh.ch"
```
We have moved almost everything already away from the Mr. Klicker Support :D Use klicker@bf.uzh.ch everywhere.